src/content/blog/posts/iceberg-athena-dbt.md

```diff
@@ -0,0 +1,116 @@
+
+Whilst Iceberg improves the reliability of Athena-based ELT pipelines, Athena queries are still SQL-based which can lead to challenges in code maintenance. This is where dbt proves to be particularly valuable.
+
+### dbt for maintainability
+
+[dbt](https://docs.getdbt.com/docs/introduction) is a transformation tool that extends SQL with features commonly associated with programming languages, enabling more flexible, scalable, and maintainable data transformation workflows. dbt is an extensible framework composed of multiple components. On the Analytical Platform we use the following components:
+
+* [dbt-core](https://github.com/dbt-labs/dbt-core), an open-source command line tool written in python  
+* [dbt-athena](https://github.com/dbt-labs/dbt-athena), a dbt-maintained trusted adapter which enables dbt to work with Athena  
+* [dbt-utils](https://github.com/dbt-labs/dbt-utils) and [dbt-audit-helper](https://github.com/dbt-labs/dbt-audit-helper), dbt-maintained packages which make it easier to apply common SQL-based functionalities and validations
+
+dbt can assure the quality of transformations through [data tests](https://docs.getdbt.com/docs/build/data-tests), for example to check whether columns contain null values. Note that with Iceberg storing much of this information as metadata, querying it through Athena becomes practically cost-free. dbt uses the concept of [threads](https://docs.getdbt.com/docs/running-a-dbt-project/using-threads) to parallelize models that have no dependencies, which speeds up runtime. This makes it easier to take advantage of Athena’s capacity so that you can run more queries within the concurrency quota and keeps queries from being cancelled for running too long.
+
+dbt allows you to combine SQL with [Jinja](https://jinja.palletsprojects.com/en/3.1.x/), a templating language, and create [macros](https://docs.getdbt.com/docs/build/jinja-macros) which are analogous to "functions" in other programming languages. Similar to functions, macros minimise code redundancy and make it easier to unit test logic. Note that whilst dbt-core v1.8 is introducing support for [unit tests](https://docs.getdbt.com/docs/build/unit-tests), this is limited to models for now. There are various community-supported dbt extensions, such as [dbt-unit-testing](https://github.com/EqualExperts/dbt-unit-testing), which can unit test macros. Nevertheless, we opted to adhere with the unit testing approach used in [dbt-utils](https://github.com/dbt-labs/dbt-utils) for its simplicity and readability.
```

Comment:
"_Note that whilst dbt-core v1.8 is introducing support for unit tests..._"

WIll this make the article age badly? Better to leave out?